.github/boring-cyborg.yml

```diff
@@ -44,6 +44,15 @@ labelPRBasedOnFilePath:
     - docs/howto/operator/snowflake/*
     - tests/providers/snowflake/**/*
 
+  area:providers:
+    - airflow/providers/**/*
+    - docs/apache-airflow-providers-*/**/*
+    - tests/providers/**/*
+    - scripts/docker-compose/ci/integration-*.yml
+    - tests/core/test_providers_manager.py
+    - scripts/in_container/run_install_and_test_provider_packages.sh
+    - scripts/in_container/run_prepare_provider_documentation.sh
```

Comment:
Let's remove them, not strictly needed

```suggestion
```

Reply:
Ok, I tried to add common files changed when adding/update providers.

Reply:
@kaxil done =)